### ISSUE TITLE
Connect to Teamspeak 5 Servers

### ISSUE THREAD

**Issue by Timon0586** [Splamy/TS3AudioBot#1021]
The Bot can't since some Teamspeak Updates connect to a Teamspeak 5 Server, maybe something was changed? The regular Teamspeak 3 Client can connect. If you need, i can  give you a Teamspeak 5 Server address for testing. and btw. can you implement that the bot reacts to pokes too? so in the Teamspeak 5 Server the chat is based on Matrix and of course a musicbot can't login to matrix but a musicbot can react to pokes.

i hope the bot can soon connect to teamspeak 5 Servers and react to pokes.

Version: 0.13.0-alpha.41

Best regards

**Comment by Flakebi** [comment]
Hi, I can connect just fine to the TS 5 server at beta.voice.teamspeak.com with the latest develop branch (which was 0.13.0-alpha.41). What is the server that you cannot connect to?
Good point about messages, I implemented pokes.

**Comment by Timon0586** [comment]
hello,
i don't know but i think beta.voice.teaamspeak.com is a rly old server. with the actually servers ( sponsored.teamspeak.com:10051 (is maybe older too) or for selfhost the actually native version - https://github.com/TeamSpeak-Systems/ts-native if you don't want to use the docker stuff ) the bot can't connect.

**Comment by Flakebi** [comment]
Thanks for the references. I pushed a fix to the `develop` branch. With that, I can connect to a recent ts-native server. Can you try again please?

**Comment by Timon0586** [comment]
i can't test it .. on the website splamy.de is the actually nightly version the beta41 from may 2022 and if i click here to "download" the develop version, i will redirect to https://splamy.de/api/nightly/ts3ab/develop_linux_x64/download and for that i got a "Too many redirects" error

**Comment by Flakebi** [comment]
Yeah, for now you need to build that version yourself if you need it. The frontend build broke (without a change) and the CI upload to the server isn’t working either unfortunately.
